telegram/constants.py

```diff
@@ -1754,3 +2587,50 @@ class ForumTopicLimit(IntEnum):
     * :paramref:`~telegram.Bot.edit_general_forum_topic.name` parameter of
       :meth:`telegram.Bot.edit_general_forum_topic`
     """
+
+
+class Date(Enum):
+    """This enum contains the date literal for :class:`telegram.InaccessibleMessage`
+
+    .. versionadded:: NEXT.VERSION
+    """
+
+    __slots__ = ()
+
+    ZERO_DATE = datetime.datetime(1970, 1, 1, tzinfo=UTC)
+    """:obj:`datetime.datetime`, value of unix 0."""
+
+
+class ReactionType(StringEnum):
+    """This enum contains the available types of :class:`telegram.Update`. The enum
+    members of this enumeration are instances of :class:`str` and can be treated as such.
+
+    .. versionadded:: NEXT.VERSION
+    """
+
+    __slots__ = ()
+
+    EMOJI = "emoji"
+    """:obj:`str`: A :class:`telegram.ReactionType` with a normal emoji."""
+    CUSTOM_EMOJI = "custom_emoji"
+    """:obj:`str`: A :class:`telegram.ReactionType` with a custom emoji."""
+
+
+class ReactionEmojis(StringEnum):
```

Comment:
```suggestion
class ReactionEmoji(StringEnum):
```

List of members is not yet complete

Reply:
Will wait with that till we have an official permission to use the description